src/gluon/tasks/packages/ConfigurePackageInJenkins.ts

```diff
@@ -54,7 +56,10 @@ export class ConfigurePackageInJenkins extends Task {
     }
 
     protected async executeTask(ctx: HandlerContext): Promise<boolean> {
-        await this.ocService.login();
+
+        const project = await this.gluonService.projects.gluonProjectFromProjectName(this.project.name);
```

Comment:
```suggestion
        const project: QMProject = await this.gluonService.projects.gluonProjectFromProjectName(this.project.name);
```

Same thing with needing to import this type as above.

Reply:
OK